src/Lean/Server/FileWorker/RequestHandling.lean

```diff
@@ -34,7 +34,9 @@ def handleCompletion (p : CompletionParams)
   -- such as a trailing dot after an option name. This shouldn't be a problem since any subsequent
   -- command starts with a keyword that (currently?) does not participate in completion.
   withWaitFindSnap doc (·.endPos + ' ' >= pos)
-    (notFoundX := pure { items := #[], isIncomplete := true }) fun snap => do
+    (notFoundX :=
+      -- work around https://github.com/microsoft/vscode/issues/155738
+      pure { items := #[{label := "-"}], isIncomplete := true }) fun snap => do
```

Comment:
Note that the the completion item should not be observable in practice as a follow-up request must already be received to end up in this branch (or when our snapshots are screwed up).